week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -120,7 +133,13 @@ function getLocationName() { }
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
-  // Implement the function body
+  let arr1 = [];
+  for (let i = 0; i < locations.length; i++) {
```

Comment:
while using loops is not wrong here. However how about using some of the functions you created before 
isAccessibleByTransportMode